pkgs/applications/misc/taskopen/default.nix

```diff
@@ -1,42 +1,49 @@
-{ lib, stdenv, fetchFromGitHub, makeWrapper, which, perl, perlPackages }:
-
-stdenv.mkDerivation rec {
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  darwin,
+  nim,
+  git,
+}:
+stdenv.mkDerivation (finalAttrs: rec {
   pname = "taskopen";
-  version = "1.1.5";
+
+  version = "v2.0.1";
 
   src = fetchFromGitHub {
-    owner = "ValiValpas";
+    owner = "jschlatow";
     repo = "taskopen";
-    rev = "v${version}";
-    sha256 = "sha256-/xf7Ph2KKiZ5lgLKk95nCgw/z9wIBmuWf3QGaNebgHg=";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Gy0QS+FCpg5NGSctVspw+tNiBnBufw28PLqKxnaEV7I=";
```

Comment:
Just a note: upstream is vulnerable to repo-jacking
Isn't an issue for Nix because we check hash (except that update bot may be vulnerable).
Also, in the future, builds referring to the old version may fail. What is sad.

If you have contact to owner, please alert him.